src/chapters/orm-relationships/one-to-many.rst

```diff
@@ -0,0 +1,198 @@
+Creating a One-to-Many Relationship
+===================================
+
+The previous section established a persistent many-to-one relationship between ``Event`` objects and ``EventCategory`` objects. This section demonstrates how to create the inverse one-to-many relationship.
+
+Setting Up the Relationship - Video
+-----------------------------------
+
+.. raw:: html
+
+   <div style="text-align:center;"><iframe width="800" height="450" src="https://www.youtube.com/embed/RLykFBY9Rys" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></div>
+
+The final code from this video is in the `one-to-many branch <https://github.com/LaunchCodeEducation/coding-events/tree/one-to-many>`__ of ``coding-events``.
+
+Setting Up the Relationship - Text
+----------------------------------
+
+Creating a one-to-many relationship from categories to events means that each ``EventCategory`` object will have access to each event that belongs to it. Near the top of ``EventCategory``, add a new field to store these events.
+
+Model Configuration
+^^^^^^^^^^^^^^^^^^^
+
+.. sourcecode:: java
+   :lineno-start: 19
+
+   private final List<Event> events = new ArrayList<>();
+
+The ``events`` field will be initialized when an ``EventCategory`` object is created. It is marked ``final`` to ensure that the collection isn't deleted or replaced. Since it is final, only a getter is needed; add one near the bottom of the class.
+
+.. admonition:: Note
+
+   Recall that when a collection is marked ``final`` its *contents* are allowed to change, but the collection object itself may not be changed. In other words, we may add or remove items from the ``events`` list, but we may not reassign the value of ``events``.
```

Comment:
Phrasing edit: Maybe, "...but we may not _reassign_ the values of any items within ``events``."

Reply:
Your suggestion sounds like the opposite of what we're trying to say (to my reading). We're trying to say that we may change the contents of `events`, but we may not reassign the variable/field itself.